plugin/templates.vim

```diff
@@ -356,9 +359,9 @@ function <SID>TExpandVars()
 	let l:camelclass = substitute(l:class, "_", "", "g")
 
 	" Define license variable
-	if executable('licensee')
+	if executable('licensee') && g:templates_use_licensee
 		" Returns 'None' if the project does not have a license.
-		let l:license = matchstr(system("licensee detect"), '^License:\s*\zs\S\+\ze\%x00')
+		let l:license = matchstr(system("licensee detect " . expand("%:p:h")), '^License:\s*\zs\S\+\ze\%x00')
```

Comment:
I think it would be better to pass the result of the `expand()` call through `shellescape()` as well:

```vim
let l:license = matchstr(system("licensee detect " . shellescape(expand("%:p:h")), ...)
```

This will avoid potential breakage when file names contain spaces, quote marks, and other “funny” characters.